dataprofiler/tests/profilers/test_unstructured_labeler_profile.py

```diff
@@ -219,7 +220,70 @@ def test_unstructured_labeler_profile_add(self, mock):
         self.assertEqual(merged_profile.times["data_labeler_predict"],
                          profile1.times["data_labeler_predict"] +
                          profile2.times["data_labeler_predict"])
-        
+
+    @mock.patch('dataprofiler.profilers.'
+                'unstructured_labeler_profile.DataLabeler')
+    @mock.patch('dataprofiler.profilers.'
+                'unstructured_labeler_profile.'
+                'CharPostprocessor')
+    def test_diff(self, mock1, mock2):
+        """
+        Tests to see that entity percentages match the counts given
+        """
+        profiler1 = UnstructuredLabelerProfile()
+        profiler1.char_sample_size = 20
+        profiler1.word_sample_size = 10
+        profiler1.entity_counts["postprocess_char_level"]["UNKNOWN"] = 6
```

Comment:
Can we try a profile that has different entity labels and also no entity labels (empty profile)?